gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -700,6 +708,11 @@ public Builder setEndpoint(String endpoint) {
       return this;
     }
 
+    public Builder setAllowHardBoundTokenTypes(ArrayList<String> allowedValues) {
```

Comment:
Do we expect customers to set this? Or only internal teams like Spanner/Storage to set it? If it is the latter, can we mark this method as `@InternalApi`?

Reply:
Right we don't want external customers to set it.

Reply:
We only expect internal teams to set this field, marked as `@InternalApi` in https://github.com/googleapis/sdk-platform-java/pull/3467/commits/e193cc9412ae68b01ca5d1a21eb38baf35d68a71